src/workerd/io/trace.h

```diff
@@ -641,4 +683,89 @@ inline SpanBuilder SpanBuilder::newChild(kj::ConstString operationName, kj::Date
       kj::mv(operationName), startTime);
 }
 
+// Lime tracing
```

Comment:
I'm not super convinced we need `LimeSpanBuilder` at all. It looks to me that `SpanBuilder` is totally sufficient for both branches of tracking. Having single builder class will simplify maintenance a lot.